osclib/stagingapi.py

```diff
@@ -524,7 +529,7 @@ def superseded_request(self, request, target_requests=None):
                 replace_old = request_old.find('state').get('name') in ['revoked', 'superseded']
 
                 if (request_new.find('action').get('type') == 'delete' and
-                    request_old.find('action').get('type') == 'delete'):
+                        request_old.find('action').get('type') == 'delete'):
```

Comment:
I don't find this beneficial or relevant to this change (nor the rest of these white-space changes).

Reply:
autopep...